Tools/validate.py

```diff
@@ -0,0 +1,78 @@
+#! /usr/bin/env python3
+""" Script to validate YAML file(s) against a YAML schema file """
+
+import argparse
+import logging
+import pprint
+import json
+import sys
+import os
+
+import yaml
+import cerberus
+
+
+def load_data_file(file_name: str, filetype: str):
+    with open(file_name) as stream:
+        if filetype == "yaml":
+            return yaml.safe_load(stream)
+        elif filetype == "json":
+            return json.load(stream)
+        else:
+            return dict()
+
+
+def main():
+    parser = argparse.ArgumentParser(description='Validate YAML or JSON file(s) against a schema')
+
+    parser.add_argument('data_file', nargs='+', help='data file(s)')
+    parser.add_argument('--schema-file', type=str, action='store',
+                        help='schema file', required=True)
+    parser.add_argument('--filetype', type=str, choices=['yaml', 'json'],
```

Comment:
Why isn't this automatically determined?

Reply:
I never thought about that, but yes I can determine it by file extension.